src/alr/alr-commands-toolchain.adb

```diff
@@ -37,7 +40,13 @@ package body Alr.Commands.Toolchain is
          Cmd.Install'Access,
          Switch      => "-i",
          Long_Switch => "--install",
-         Help        => "Install a toolchain component");
+         Help        => "Install one or more toolchain component(s)");
```

Comment:
I think the "(s)" shouldn't be there when "one or more" is there too. Likewise in the spec file.